CHANGELOG.md

```
@@ -7,6 +7,21 @@ and Yorkie adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
 ## [Unreleased]
 
+## [v0.6.41] - 2025-12-09
+
+### Added
+
+- Introduce GetRevision RPC and fix revision restore by @hackerwins in https://github.com/yorkie-team/yorkie/pull/1595
+
+### Changed
+
+- Replace Bitnami MongoDB chart with Percona Operator by hackerwins in https://github.com/yorkie-team/yorkie/pull/1594
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)